src/MongoDB/BulkWrite.c

```diff
@@ -391,26 +390,24 @@ static PHP_METHOD(MongoDB_Driver_BulkWrite, insert)
 	Z_PARAM_ARRAY_OR_OBJECT(zdocument)
 	PHONGO_PARSE_PARAMETERS_END();
 
-	bson_flags |= PHONGO_BSON_RETURN_ID;
```

Comment:
This is consistent with the newer code going into `BulkWriteCommand.c`. There was really no need for a variable here.